hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILECrck.py

```diff
@@ -108,14 +115,15 @@ def get_data_module(self) -> TilesDataModule:
             ]
         )
         return TcgaCrckTilesDataModule(
-            root_path=self.local_datasets[0],
+            root_path=self.local_dataset,
             max_bag_size=self.max_bag_size,
             batch_size=self.batch_size,
             transform=transform,
             cache_mode=self.cache_mode,
+            precache_location=self.precache_location,
             cache_dir=self.cache_dir,
-            # number_of_cross_validation_splits=self.number_of_cross_validation_splits,
-            # cross_validation_split_index=self.cross_validation_split_index,
+            number_of_cross_validation_splits=self.number_of_cross_validation_splits,
```

Comment:
The run will fail currently since cross val has not been added yet - if you want it to run, you will need to comment these 2 lines out